racingcar-java/src/main/java/racingcar/Cars.java

```diff
@@ -0,0 +1,42 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class Cars {
+
+    final List<Car> carList = new ArrayList<>();
```

Comment:
```suggestion
    private final List<Car> cars = new ArrayList<>();
```
접근제어자 확인부탁드려요~
리스트와같은 자료구조를 표현할때는 자료구조를 표현해주기보다는 복수형으로 변수명을 나타내는데요. 둘을 차이가 무엇일까요?

Reply:
생각해봤는데 잘 모르겠습니다.
ArrayList나 Vector 같은 구현체의 차이인가요?

Reply:
아..! 진짜 '리스트' 일 수도 있겠군요

이해 했습니다!!
조언 감사합니다 :)